src/components/MobileMenuBar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
+import menu from '../assets/menu.svg'
+import { mediaQueries } from './Common';
+
+const MobileMenuBarContainer = styled.div`
+  display: none; 
+  padding: 20px;
+  ${[mediaQueries[0]]} {
+    display: inline-block;
+    text-align: center;
+    width: 100%;
```

Comment:
If we are using 100% make sure to include `box-sizing: border-box`, else the menu will be wider than the screen size

Reply:
good tip, TIL